config/test.exs

```diff
@@ -6,4 +6,5 @@ config :bolt_sips, Bolt,
   port: 7687,
   pool_size: 5,
   max_overflow: 1
+
 z
```

Comment:
Consider removing this unwanted character

Reply:
Good catch, Thanks!

Reply:
done